packages/dds/tree/src/simple-tree/core/treeNodeKernel.ts

```diff
@@ -181,6 +188,15 @@ export class TreeNodeKernel {
 			// Hydrated case
 			this.#hydrationState = this.createHydratedState(innerNode.anchorNode);
 			this.#hydrationState.innerNode = innerNode;
+
+			// Forward relevant anchorNode events to our event handler
+			for (const eventName of kernelEvents) {
+				this.#hydrationState.offAnchorNode.add(
+					innerNode.anchorNode.events.on(eventName, (arg) =>
+						this.#events.value.emit(eventName, arg),
```

Comment:
This doesn't affect the timing of the demand of the lazy events object, does it? I'm trying to puzzle exactly what the benefit of the laziness is. Looking at the code above, it seems like it's going to be demanded whenever an event fires on this node - even if there are no listeners. Is that helpful? I'm confused. I don't think it matters much here. But I'm confused.

Reply:
If this PR isn't making it any _more_ eager (which I don't think it is), then just leave it.

Low priority, but in the future I feel like we should try to make it more lazy - it shouldn't demand until any listeners are registered.